mongo/options/clientoptions.go

```diff
@@ -99,6 +99,7 @@ type ClientOptions struct {
 	AutoEncryptionOptions    *AutoEncryptionOptions
 	ConnectTimeout           *time.Duration
 	Compressors              []string
+	Crypt                    driver.Crypt
```

Comment:
Is it possible to move `Crypt` to an internal type?

Reply:
No need to make `Crypt` internal since there doesn't seem to be a clear alternative. I suggest following suit with `Deployment` or `AuthenticateToAnything` and marking in internal.